i18n/ko.json

```diff
@@ -670,7 +670,7 @@
   "admin.jobTable.statusInProgress": "진행 중",
   "admin.jobTable.statusPending": " 보내는 중",
   "admin.jobTable.statusSuccess": "성공",
-  "admin.ldap.baseDesc": "The Base DN is the Distinguished Name of the location where Mattermost should start its search for users in the LDAP tree.",
+  "admin.ldap.baseDesc": "The Base DN is the Distinguished Name of the location where Mattermost should start its search for user and group objects in the AD/LDAP tree.",
```

Comment:
Only the original English translations need to be provided for the Webapp repo, the rest should automatically get updated by https://translate.mattermost.com/. 🙂

So, no need to include this in the PR.

Reply:
That said, I do see the discrepancy between the en.json and ko.json files before this PR. @enahum, should we be concerned that this entry in en.json and ko.json were different prior to this PR?

Reply:
that one will get included automatically if it does not get translated in the translation server. the only one thing to do in this PR is remove the changes in this file.

Reply:
Thanks for pointing this out! Done! 😄